lib/omniauth/strategies/recharge.rb

```diff
@@ -7,7 +7,7 @@ class Recharge < OmniAuth::Strategies::OAuth2
       option :provider_ignores_state, false
 
       option :client_options, {
-          :site          => 'https://shopifysubscriptions.com',
+          :site          => 'https://admin.rechargeapps.com/partners/app',
```

Comment:
i don't know why this change is here, i did not make it

Reply:
![image](https://user-images.githubusercontent.com/16980768/123789992-b6b6d080-d8d5-11eb-9106-18223bb95221.png)